src/toplevel.c

```diff
@@ -88,12 +88,15 @@ jl_value_t *jl_eval_module_expr(jl_expr_t *ex)
 {
     static arraylist_t module_stack;
     static int initialized=0;
+    jl_module_t *last_module;
```

Comment:
stylistically, this should have an initializer here (although you can just drop this file from the change set)